week3/homework/src/server.js

```diff
@@ -0,0 +1,55 @@
+'use strict'
+
+const express = require('express');
+const todoMethodList = require('./index');
+const fs = require('fs');
+const bodyParser = require('body-parser');
+
+const app = express();
+app.use(bodyParser());
+
+const port = 3000;
+app.get('/todos', (req, res) => {
+  res.send(todoMethodList.readToDoFile());
+});
+app.get('/todos/:id', (req, res) => {
+  res.send(todoMethodList.readToDoFile(req.params.id));//todoMethodList[0](req.params.id));
```

Comment:
It currently doesn't return the todo, but by first converting the id to an index it should work:
`res.send(todoMethodList.readToDoFile(todoMethodList.findIndexTodo(req.params.id)));`
Right?